chapters/external-repository-identifiers.md

```diff
@@ -238,15 +238,19 @@ Examples:
 
 External Reference Site: [Software Heritage persistent IDentifiers](https://docs.softwareheritage.org/devel/swh-model/persistent-identifiers.html)
 
-### F.4.1 gitbom <a name="gitbom"></a>
+### F.4.2 gitbom <a name="gitbom"></a>
 
-Locator Format: gitoid":"\<git object type>":"\<hash algorithm>":"\<hash value>
-	
-Contextual Example: gitoid:blob:sha1:261eeb9e9f8b2b4b0d119366dda99c6fd7d35c64
+Locator Format: `gitoid":"\<git object type>":"\<hash algorithm>":"\<hash value>`
+
+Locator Format Reference: <https://www.iana.org/assignments/uri-schemes/prov/gitoid>
```

Comment:
While an external reference is useful, we should have the details in this document:

- Valid values for <git object type> are `blob`, `tree`, `commit`, and `tag`.
- Valid values for <hash algorithm> are 'sha1' and 'sha256'
- <hash value> should be expressed as a hexadecimal string in lower case.

Reply:
That's a good point. I'll update.